### PR TITLE
Fail Travis / Docker build if Jsonnet does not run

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Jsonnet currently fails silently

### How to review 
Are census schemas built in the container?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,11 @@
 FROM ubuntu:18.04 as builder
 
 RUN apt-get update \
-    && apt-get install -y curl unzip
+    && apt-get install -y curl unzip git make build-essential
```
Comment (ajmaddaford) [inline Dockerfile:4]: Are curl and unzip needed?

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jonnyshaw89) [reply Dockerfile:4]: yeh for `load_templates.sh`

File: .gitignore [hunk 2]
```diff
@@ -92,7 +90,7 @@ requirements.txt
 .screenshots
 
 # Ignore built schemas
-data/en/census_individual.json
+/data
```
Comment (bitdivision) [inline .gitignore]: Should this have a leading slash?

Comment (jonnyshaw89) [reply .gitignore]: yes

Comment (jonnyshaw89) [reply .gitignore]: no

**Review by bitdivision — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**